recipe/0001-fix-tuple-in-geos.patch

```diff
@@ -0,0 +1,11 @@
+--- shapely/geos.py	2019-12-31 18:31:44.000000000 +0100
++++ shapely/geos.py	2020-03-23 08:48:10.430911180 +0100
+@@ -221,7 +221,7 @@
+                 # finishGEOS.
+                 new_func.argtypes = [c_void_p]
+             else:
+-                new_func.argtypes = [c_void_p] + old_func.argtypes
++                new_func.argtypes = [c_void_p] + list(old_func.argtypes)
```

Comment:
It doesn't seem like this should be necessary, but I'm patching to see what the next issues are.

Reply:
Hmm, a quick test shows that adding a `tuple` and a `list` in either order is not allowed in either `pypy` or `CPython`.  Odd that this only comes up in `pypy`.